File: 2022-2023/sourcecode/lab/lab11/RetrofitPostBasicsJava/app/src/main/java/pl/udu/uwr/pum/retrofitpostbasicsjava/Post.java
package pl.udu.uwr.pum.retrofitpostbasicsjava;

import com.google.gson.annotations.SerializedName;

public class Post {
    private final Integer userId;
    private Integer id;
    private final String title;

    @SerializedName("body")
    String content;

    public Post(Integer userId, String title, String content) {
        this.userId = userId;
        this.title = title;
        this.content = content;
    }

    public int getUserId() {
        return userId;
    }

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getContent() {
        return content;
    }
}

File: 2022-2023/sourcecode/lab/lab11/RetrofitPostBasicsKotlin/app/src/main/java/pl/udu/uwr/pum/retrofitpostbasicskotlin/Post.kt
package pl.udu.uwr.pum.retrofitpostbasicskotlin

import com.google.gson.annotations.SerializedName

data class Post (
    val userId: Int,
    val title: String,

    @SerializedName("body")
    val content: String
){
    var id: Int? = null
}
